doc/source/reference/cluster.rst

```diff
@@ -2,9 +2,3 @@
    :no-members:
    :no-inherited-members:
    :no-special-members:
-
-.. toctree::
-   :hidden:
-
-   cluster.vq
-   cluster.hierarchy
```

Comment:
It's arguable that we want these in the main list because `cluster` has no other attributes. I'll let @stefanv confirm because he asked about this before, but I'm OK with either.